cypress/integration/BonnieFHIR/WebInterface/Patient/DragAndDropDatePopulation.spec.js

```diff
@@ -0,0 +1,86 @@
+import * as helper from '../../../../support/helpers'
+import * as bonnieLogin from '../../../../support/BonnieFHIR/BonnieLoginLogout'
+import * as measureDetailsPage from '../../../../pom/BonnieFHIR/WI/MeasureDetailsPage'
+import * as deleteMeasure from '../../../../support/BonnieFHIR/DeleteMeasure'
+import * as testPatientPage from '../../../../pom/BonnieFHIR/WI/TestPatientPage'
+import * as bonnieUploadMeasure from '../../../../support/BonnieFHIR/BonnieUploadMeasure'
+
+const measureName = 'FHIRmeasureCMS347'
+const measureFileToUpload = 'FHIRmeasureCMS347-v0-0-003-FHIR-4-0-1.zip'
+
+const lastNameSuffix = new Date().getTime()
+const distinctLastName = 'President' + lastNameSuffix
+
+const expectedStartDate = '01/01/2019'
+const expectedStartTime = '11:00 AM'
+const expectedEndDate = '01/01/2019'
+const expectedEndTime = '12:00 PM'
+
+describe('Patient: Elements', () => {
+
+  before('Login', () => {
+
+    bonnieLogin.login()
+    bonnieUploadMeasure.UploadMeasureToBonnie(measureFileToUpload, false)
+    measureDetailsPage.navigateToMeasureDetails(measureName)
+
+  })
+  after('Log Out', () => {
+
+    deleteMeasure.DeleteMeasure(measureName)
+    helper.visibleWithTimeout(measureDetailsPage.measurePageNavigationBtn)
+    bonnieLogin.logout()
+
+  })
+
+  it('Verify Drag and Drop date population', () => {
+
+    cy.get(measureDetailsPage.patientListing).then((patientListing) => {
+      const initialPatientCount = parseInt(patientListing.text())
```

Comment:
Codacy complains about missing base although it's not an issue in this case. 
`parseInt(patientListing.text(), 10) `